signing/src/main/java/tech/pegasys/web3signer/signing/BlsKeystoreBulkLoader.java

```diff
@@ -39,36 +39,59 @@
 public class BlsKeystoreBulkLoader {
   private static final Logger LOG = LogManager.getLogger();
 
-  public Collection<ArtifactSigner> loadKeystoresUsingPasswordDir(
+  public SecretValueResult<ArtifactSigner> loadKeystoresUsingPasswordDir(
```

Comment:
This doesn't seem like the right type to expose, we are not returning the secret values but artifacts. Can we create a type to hold the collection of artifactSigners and the number of errors.

Reply:
But this is the same type which is used by Aws bulkloading and Azure bulkloading as well.